tests/pipelines/kitctl/run.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: pipeline-template
+  namespace: tekton-pipelines
+spec:
+  params:
+  - name: name
+    value: "example"
+  - name: substrate-name
+    value: "YOUR_SUBSTRATE_NAME"
+  podTemplate:
+    nodeSelector:
+      kubernetes.io/arch: amd64
+  serviceAccountName: tekton-pipelines-executor
```

Comment:
Is it possible to use a default?

Reply:
The Tekton runs use the default serviceaccount for the namespace of the run object. That account is lacking the RBAC permissions to perform tasks defined by the tasks. Rather than given strong RBAC permissions to the default account, I think will be better to create a dedicated service account with just the right amount of permission.